specification/datashare/resource-manager/Microsoft.DataShare/stable/2020-09-01/DataShare.json

```diff
@@ -0,0 +1,6755 @@
+          "type": "string"
+        },
+        "sqlServerResourceId": {
+          "description": "Resource id of SQL server",
+          "type": "string"
+        },
+        "tableName": {
+          "description": "SQL DW table name.",
+          "type": "string"
+        }
+      }
+    },
+    "SynapseWorkspaceSqlPoolTableDataSet": {
+      "description": "A Synapse Workspace Sql Pool Table data set.",
+      "required": [
+        "properties",
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataSet"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SynapseWorkspaceSqlPoolTableDataSetProperties",
+          "description": "Synapse Workspace Sql Pool Table data set properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "SynapseWorkspaceSqlPoolTable"
+    },
+    "SynapseWorkspaceSqlPoolTableDataSetMapping": {
+      "description": "A Synapse Workspace Sql Pool Table data set mapping",
+      "required": [
+        "properties",
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataSetMapping"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SynapseWorkspaceSqlPoolTableDataSetMappingProperties",
+          "description": "A Synapse Workspace Sql Pool Table data set mapping properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "SynapseWorkspaceSqlPoolTable"
+    },
+    "SynapseWorkspaceSqlPoolTableDataSetMappingProperties": {
+      "description": "Properties of the Synapse Workspace SQL Pool Table data set mapping",
+      "required": [
+        "synapseWorkspaceSqlPoolTableResourceId",
+        "dataSetId"
+      ],
+      "type": "object",
+      "properties": {
+        "dataSetId": {
+          "description": "The id of the source data set.",
+          "type": "string"
+        },
+        "dataSetMappingStatus": {
+          "description": "Gets the status of the data set mapping.",
+          "enum": [
+            "Ok",
+            "Broken"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "dataSetMappingStatus",
+            "modelAsString": true
+          }
+        },
+        "provisioningState": {
+          "description": "Provisioning state of the data set mapping.",
+          "enum": [
+            "Succeeded",
+            "Creating",
+            "Deleting",
+            "Moving",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        },
+        "synapseWorkspaceSqlPoolTableResourceId": {
```

Comment:
Please review all the new properties and mark them as readOnly if the value is meant to be set by your service only.